config/testnet/gas.toml

```diff
@@ -0,0 +1,13 @@
+[gas]
+
+[gas.flows.compact_resource_lock]
+# Gas units captured by scripts/e2e/estimate_gas_compact.sh
+open = 0
+fill = 76068
+claim = 121995
+
+[gas.flows.permit2_escrow]
```

Comment:
We're missing 3009_escrow gas prices here. Would you mind adding them to both testnet and demo configs (make sure to update deployments.sh and setup_testnet.sh)? You can get the values from ./oif-demo intent test escrow 3009 A2B and checking the tx receipts with forge for gas units used I believe.

Reply:
Or add a script in e2e similar to the estimate scripts we already have..